### DISCUSSION TITLE
My request was denied.

### DISCUSSION THREAD

**Discussion by Bronty-X** [community/community#51247] [GitHub Education]
- I'm a correspondence high school student, so I indicated that I was in a correspondence program, as you can see in the picture, but they wouldn't accept me.
- I have updated my profile according to the instructions and have applied multiple times but have not been accepted.
If there is any way I can do this, please let me know.
- I am a Japanese student and could only prepare documents in Japanese. However, even though I provided my translated name and expiration date, it was never approved.
![image](https://user-images.githubusercontent.com/62980663/228288618-903e196a-d07e-4f00-b24d-cc8dc6f55ced.png)

**Comment by samus-aran** [comment]
Hi @Bronty-X 

We're sorry to hear of your frustration. As we direct users to utilise the community more and share knowledge on all things GitHub Global Campus sometimes, like this, users require explicit help. But we're here to assist ⭐

Here is a quick tip on submitting evidence:

- If your original document is not in English then we advise you to take a photo of the original next to an English translation. This also include your profile information.
- Ensure your evidence correspond to details in your GitHub profile. The verification process looks at your GitHub username, email and name ( as well as other points) to cross reference the evidence submitted. Do you use a variation of your name or a nickname?  It’s these small but important points to consider when applying that can make a much smoother process.

We would also suggest that when you update your profile information that you log out and log back into GitHub before re-applying.

Ensure you select the most appropriate **evidence type** in the dropdown. The order of options indicates the preference of evidence with Academic Email being at the top. If you don't have an Academic email or Student ID, do try and acquire evidence that is positioned higher rather than lower in that option list.

If you have been repetitively unsuccessful using a particular evidence type (e.g. Student ID) we would recommend that you acquire and submit an alternative piece of evidence that can verify your student status.

Hope this helps 👍
